### PR TITLE
Continue Pr/1250

### LINKED ISSUE #1213: Scrollbar is light in dark mode
**Describe the bug**

Scrollbar is light even in dark mode

**To Reproduce**
1. Open long pages Ex: https://snapshot.org/#/olympusdao.eth
2. Check the scrollbar

**Expected behavior**

The scrollbar should be dark

**Screenshots**

image

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome
 - Version Beta

### PR DESCRIPTION
**Pull request by samuveth**
Fixes #1213

### CODE REVIEW

File: src/style.scss
Comment (bonustrack) [inline src/style.scss:17]: Can't we just add `color-scheme: light;` here?

Comment (samuveth) [reply src/style.scss:17]: No, couldn't get that to work.

File: src/composables/useUserSkin.ts
Comment (samuveth) [inline src/composables/useUserSkin.ts:49]:
```suggestion
  function toggleColorScheme() {
    document.documentElement.setAttribute(
      'data-color-scheme',
      userSkin.value === LIGHT_MODE ? 'light' : 'dark'
    );
  }

  watch(userSkin, () => {
    toggleColorScheme();
  });

  onMounted(() => toggleColorScheme());
```

File: src/composables/useUserSkin.ts
Comment (samuveth) [inline src/composables/useUserSkin.ts:24]:
```suggestion
```

File: src/composables/useUserSkin.ts
Comment (samuveth) [inline src/composables/useUserSkin.ts:20]:
```suggestion
```

File: src/style.scss
Comment (mktcode) [reply src/style.scss:17]: I also don't see any better way. If the dark mode shall be applied to the main browser window's scrollbar, the root element (`<html>`) needs some class/attribute, which requires this custom piece of code in the watcher.

Just one thing: With this we don't need `.light-mode` and `.dark-mode` on the app div.

**Review by mktcode — COMMENTED** [review]
See comment on above conversation.

File: src/style.scss
Comment (samuveth) [reply src/style.scss:17]: > Just one thing: With this change, `.light-mode` and `.dark-mode` on the app div are redundant now. Maybe this could be cleaned up before merge.

Are they really, I'm not sure I understand you but these variables are still used.

Comment (mktcode) [reply src/style.scss:17]: Well.... I just see this:

image

Comment (mktcode) [reply src/style.scss:17]: `[data-color-scheme='light'] #app` is the same as `.light-mode`. That's what I mean.

**Comment by samuveth** [comment]
@mktcode I see, I will try this.

**Comment by samuveth** [comment]
I couldn't find any way that `eth-dark-mode` is still being used so I removed it. Maybe someone can double check this before approving.

**Comment by mktcode** [comment]
> I couldn't find any way that `eth-dark-mode` is still being used so I removed it. Maybe someone can double check this before approving.

I only found this:
https://github.com/snapshot-labs/snapshot/blob/3502f5bb8a62681f96f7c941fe3d4ef5556617c3/src/App.vue#L34
But there's no skin called "eth" anymore. It was removed here: https://github.com/snapshot-labs/snapshot-spaces/compare/f2e5daf075014fb03981061d2be52b37a661ee7c...19d86e9abaf9481ec40cc89d11b661c63c13385d

**Comment by samuveth** [comment]
> Btw... This is also a good moment to get rid of this duplicate html id:

Yes I was also thinking about that, any idea what we could change the name of this id into? https://github.com/snapshot-labs/snapshot/blob/acaf3ce871e1988c5915b5a02319f3ac617c4c48/src/App.vue#L52

**Comment by mktcode** [comment]
I think it can be removed entirely.

The `min-height` in `style.scss` can be moved:

```diff
// style.scss
html,
body,
#app {
  background: var(--bg-color);
  color: var(--text-color);
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
+  min-height: 100vh;
}

-#app {
-  min-height: 100vh;
-}
```
And this div just gets a `min-h-screen` class:
```diff
// App.vue
-<div :class="skin" id="app" class="overflow-hidden pb-6 font-serif text-base">
+<div :class="skin" class="overflow-hidden pb-6 font-serif text-base min-h-screen">
```

**Comment by samuveth** [comment]
@mktcode just removing the duplicate `id` seems to have been enough. Or do you still see the need for those other additions?

**Comment by samuveth** [comment]
@mktcode done, please approve if there isn't anything else.

**Review by bonustrack — APPROVED** [review]
Seem to work well

File: src/style.scss
Comment (mktcode) [inline src/style.scss]: Shouldn't this be `min-height`?

**Pull request merged**
